dispatch/api/validators.py

```diff
@@ -48,3 +48,11 @@ def __call__(self, slug):
         else:
             if self.model.objects.filter(slug=slug).exclude(parent=self.instance.parent).exists():
                 raise ValidationError('%s with slug \'%s\' already exists.' % (self.model.__name__, slug))
+
+class TypeValidator(object):
```

Comment:
This actually doesn't need to be a class. I would rename it to `AuthorValidator` and structure it like so:

```py
def AuthorValidator(data):
  # Check that data['person'] and data['type'] is set
```